.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a bug report
+title: "[BUG]: "
+labels: kind/bug
+assignees: ''
+
+---
+
+<!-- MAKE SURE TO APPLY SUITABLE LABELS BEFORE FILING THE ISSUE, ESPECIALLY PRIORITY -->
```

Comment:
No, the reporter should not care about what labels the repo happens to use.

Reply:
I agree, we should try make it as easy as possible to file a bug reprot

Reply:
Whoops, forgot we might not be the only ones making bug reports
```suggestion
```